src/WordPress/ClientActions.php

```diff
@@ -64,4 +81,33 @@ public function returnWordPressDomain()
 
         return $cf_zones_list;
     }
+
+    private function cacheDomainName($response)
+    {
+        // Check if domain name needs to cached
+        $wpDomain = $this->wordpressAPI->getOriginalDomain();
+        $cachedDomainList = $this->wordpressAPI->getDomainList();
+        $cachedDomain = $cachedDomainList[0];
+
+        if (Utils::getRegistrableDomain($wpDomain) !== $cachedDomain) {
+            // Since we may not be logged in yet we need to check the credentials being set
+            if ($this->wordpressClientAPI->isCrendetialsSet()) {
```

Comment:
Since this is part of the hook for GET /zones now this will never fire if they aren't logged in anymore right?  I think we can remove this check and `isCredentialsSet()`.